catalog/dags/common/cloudwatch.py

```diff
@@ -48,6 +49,11 @@ def enable_alarm_actions(self, alarm_name, enable):
 
 
 def enable_or_disable_alarms(enable):
+    skip_toggling = Variable.get("SKIP_TOGGLING_CLOUDWATCH_ALARMS", False)
+    if skip_toggling:
+        logger.info("Skipping toggling CloudWatch alarms.")
+        return
```

Comment:
Can we raise an `AirflowSkipException` so that the task is actually skipped, instead of just logging?

Note that we'd also need to update the `trigger_rule` for the tasks immediately downstream (`trigger_ingest_upstream` and `trigger_delete_index`, I think) to be `TriggerRule.NONE_FAILED` so they don't also get skipped.

Reply:
Done ✅